src/FSharpx.Collections/Collections.fs

```diff
@@ -230,6 +230,13 @@ module Seq =
               notFirst := true
       } 
 
+    /// The catOptions function takes a list of Options and returns a seq of all the Some values.
+    let catOptions (xs:seq<Option<'a>>) =
+        seq {
```

Comment:
none, the point is to have one function that does this, so that you don't have to compose two other to do the job (like in Haskell [catMaybes](http://hackage.haskell.org/package/base/docs/Data-Maybe.html#v:catMaybes))